### PR TITLE
PF-30 Add resource manager get project method

### PR DESCRIPTION
**Pull request by wchamber**

### CODE REVIEW

File: common/src/main/java/bio/terra/cloudres/common/CloudOperation.java [hunk 1]
```diff
@@ -5,6 +5,9 @@
   // Google Resource Manager Operations
   GOOGLE_CREATE_PROJECT,
   GOOGLE_DELETE_PROJECT,
+  GOOGLE_GET_PROJECT,
+  GOOGLE_GET_IAM_POLICY_PROJECT,
```
Comment (yonghaoy) [inline common/src/main/java/bio/terra/cloudres/common/CloudOperation.java:9]: nit, seems we are not using IAM enum now?

File: google-cloudresourcemanager/src/test/java/bio/terra/cloudres/google/cloudresourcemanager/CloudResourceManagerCowTest.java [hunk 2]
```diff
@@ -47,6 +51,10 @@ public void createDeleteProject() throws Exception {
             operationCow, Duration.ofSeconds(5), Duration.ofSeconds(30));
     assertTrue(operationCow.getOperation().getDone());
     assertNull(operationCow.getOperation().getError());
+
+    Project project = managerCow.projects().get(projectId).execute();
+    assertEquals(projectId, project.getProjectId());
+
     managerCow.projects().delete(projectId).execute();
```
Comment (yonghaoy) [inline google-cloudresourcemanager/src/test/java/bio/terra/cloudres/google/cloudresourcemanager/CloudResourceManagerCowTest.java:58]: do we need to assert project is deleted?

**Review by yonghaoy — APPROVED** [review]

File: common/src/main/java/bio/terra/cloudres/common/CloudOperation.java (hunk 1 above)
Comment (wchamber) [reply common/src/main/java/bio/terra/cloudres/common/CloudOperation.java:9]: Yeah, I'm batching these so I don't have to keep bunching versions for common and its dependencies. used in next PR.

File: google-cloudresourcemanager/src/test/java/bio/terra/cloudres/google/cloudresourcemanager/CloudResourceManagerCowTest.java (hunk 2 above)
Comment (wchamber) [reply google-cloudresourcemanager/src/test/java/bio/terra/cloudres/google/cloudresourcemanager/CloudResourceManagerCowTest.java:59]: Good point, it's not removed, but it does change its life cycle to `DELETE_REQUESTED`. Updated test, thanks.

**Pull request merged**